packages/core/src/components/Table/context/TableContext/TableContext.types.ts

```diff
@@ -0,0 +1,17 @@
+import { ITableProps } from "../../Table/Table";
+import React from "react";
+
+export interface ITableContext {
```

Comment:
Are these supported to be exported as well? If so, they need to be registered in Table/index.ts